api/src/controllers/kots/KotsAPI.ts

```diff
@@ -219,6 +192,18 @@ export class KotsAPI {
         await kotsAppDownloadUpdates(updatesAvailable, app, request.app.locals.stores);
 
         await request.app.locals.stores.kotsAppStore.clearUpdateDownloadStatus();
+
+        if (deploy) {
```

Comment:
UI will show this, so maybe add a status and do this before calling `clearUpdateDownloadStatus`